.env-test.template

```diff
@@ -0,0 +1,6 @@
+HUBSPOT_API_KEY=
```

Comment:
Should `.env` files be placed into `node` directory? I think they may differ depending on language

Reply:
stripe has general .env for all languages. Didn't see envs in other languages, but seems it would be a good idea to have only one env instead of having different ones per language. But currently we can move to node as temporary or permanent solution.

Reply:
Hm, okay, let's leave it as it is for now